openapi/components/schemas/Customer.yaml

```
@@ -43,8 +43,12 @@ properties:
   defaultPaymentInstrument:
     type: object
     nullable: true
-    allOf:
-      - $ref: ./PaymentInstruments/ValueObjects/PaymentInstrumentValueObject.yaml
+    description: Default payment instrument information.
```

Perhaps here should be the same description then?

```suggestion
    description: Payment instrument information.
```

---

Customer has default payment instrument